### ISSUE TITLE
`daml init` should create the daml/Main.daml file

### ISSUE THREAD

**Issue by StephenOTT** [digital-asset/daml#14048]
### What is the problem you want to solve?

when you run `daml init` it creates a single daml.yaml file.  But the yaml file references a Main.daml file that is not created.

### What is the solution you would propose?

Create the daml/Main.daml file with a **simple** starter template.  the yaml already references parties alice and bob.

If `init` is not supposed to be used as a jump-start, and it is supposed to be a empty skeleton, then it would make more sense to remove the parties from the yaml and create the Main.daml file.

**Comment by stefanobaghino-da** [comment]
@ray-roestenburg-da Makes sense, added to the backlog. Evaluate whether this could be a gardening topic.
